lib/oracledb.js

```diff
@@ -87,6 +87,122 @@ function extend(oracledb) {
         value: 0,
         enumerable: true
       },
+      DBTYPE_VARCHAR: {
+        value: 1001,
```

Comment:
Maybe use the OCI values?  Or 1000+the OCI value. One reason the node-oracledb type values start at 2000 is to avoid clashing with OCI type numbers